### ISSUE TITLE
Revealer fieldtype shows label redundantly.

### ISSUE THREAD

**Issue by theblacksheepltd** [statamic/cms#2468]
## Bug Description
In revealer fieldtype documentation, sample shows as a button with the display property used as the label. In CP, both the button and a standard field label are displayed. It seems that for its intended purpose, the label should not display.

## How to Reproduce
Add revealer field to blueprint or fieldset and set display property.

## Environment

**Statamic version:** 3.0.*

**PHP version:** 7.3

**Install method (choose one):**
- Fresh install from `statamic/statamic`

**Comment by damcclean** [comment]
I'm unable to reproduce this issue.

I've copied the field configuration from the Statamic documentation:

```yaml
  -
    handle: revealed
    field:
      type: text
      display: 'I am revealed!'
      if:
        behold: 'equals true'
  -
    handle: behold
    field:
      type: revealer
      display: 'Behold!'
```

With that configuration, I can't see the field label of the revealer field, see this [screencast](https://drive.google.com/file/d/1NZ1foWELsKjzJFLEuQGsL-nOlSuWOCOy/view?usp=sharing).

I'm going to close this issue and presume it's been fixed. If you encounter this issue again, reply on this issue with a short screencast and I can have another look.

**Comment by theblacksheepltd** [comment]
Sorry — I missed a step in my reporting. This happens when used in a
replicator or bard field. It does work fine outside of those.

Thanks,


*Brian Shepherd*
——————————
*Black Sheep Design Ltd.*
720.201.9764
theblacksheepltd.com

**Comment by damcclean** [comment]
Thanks for providing more information. I’ll re-open this issue.

**Comment by robdekort** [comment]
Possibly linked to #2289

**Comment by jackmcdade** [comment]
Fixed in 255ac140a749ff6e34d58ea24e3a301c7704d271
